package.json

```diff
@@ -100,11 +102,11 @@
     "sass-loader": "^10.1.1",
     "server-destroy": "^1.0.1",
     "sluggo": "^0.3.0",
-    "stylelint": "^13.13.0",
-    "stylelint-config-standard": "^20.0.0",
-    "stylelint-declaration-strict-value": "^1.7.12",
-    "stylelint-order": "^4.1.0",
-    "stylelint-webpack-plugin": "^2.1.1",
+    "stylelint": "^14.6.1",
```

Comment:
using `stylelint@14` fixes `@stylelint/postcss-markdown` warning

Reply:
**before**

```sh
➜  apostrophe git:(2560-npm-warnings) ✗ npx stylelint modules/**/*.{scss,vue} 

modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBarLocale.vue
 312:3  ✖  Expected variable, function or keyword for "10px" of "font-size"   scale-unlimited/declaration-strict-value
 313:3  ✖  Expected "color" to come before "font-size"                        order/properties-order                  

modules/@apostrophecms/login/ui/apos/components/TheAposLoginHeader.vue
  92:5   ✖  Expected variable, function or keyword for "#F8F9FA" of "color"    scale-unlimited/declaration-strict-value
  95:18  ✖  Expected a trailing semicolon                                      declaration-block-trailing-semicolon    
  99:7   ✖  Expected variable, function or keyword for "21px" of "font-size"   scale-unlimited/declaration-strict-value
 101:5   ✖  Unexpected empty line before closing brace                         block-closing-brace-empty-line-before   

modules/@apostrophecms/ui/ui/apos/components/AposSpinner.vue
 113:5  ✖  Unexpected vendor-prefix "-webkit-transform"   property-no-vendor-prefix
 115:5  ✖  Expected indentation of 2 spaces               indentation              
 116:5  ✖  Unexpected vendor-prefix "-webkit-transform"   property-no-vendor-prefix
 122:5  ✖  Unexpected vendor-prefix "-webkit-transform"   property-no-vendor-prefix
 124:5  ✖  Expected indentation of 2 spaces               indentation              
 125:5  ✖  Unexpected vendor-prefix "-webkit-transform"   property-no-vendor-prefix
```

**after**

```sh
➜  apostrophe git:(2560-npm-warnings) npx stylelint modules/**/*.{scss,vue} 

modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBarLocale.vue
 312:3  ✖  Expected variable, function or keyword for "10px" of "font-size"  scale-unlimited/declaration-strict-value
 313:3  ✖  Expected "color" to come before "font-size"                       order/properties-order                  

modules/@apostrophecms/login/ui/apos/components/TheAposLoginHeader.vue
  92:5   ✖  Expected variable, function or keyword for "#F8F9FA" of "color"   scale-unlimited/declaration-strict-value
  95:18  ✖  Expected a trailing semicolon                                     declaration-block-trailing-semicolon    
  99:7   ✖  Expected variable, function or keyword for "21px" of "font-size"  scale-unlimited/declaration-strict-value
 101:5   ✖  Unexpected empty line before closing brace                        block-closing-brace-empty-line-before   

modules/@apostrophecms/ui/ui/apos/components/AposSpinner.vue
 113:5  ✖  Unexpected vendor-prefix "-webkit-transform"  property-no-vendor-prefix
 115:5  ✖  Expected indentation of 2 spaces              indentation              
 116:5  ✖  Unexpected vendor-prefix "-webkit-transform"  property-no-vendor-prefix
 122:5  ✖  Unexpected vendor-prefix "-webkit-transform"  property-no-vendor-prefix
 124:5  ✖  Expected indentation of 2 spaces              indentation              
 125:5  ✖  Unexpected vendor-prefix "-webkit-transform"  property-no-vendor-prefix
```

Reply:
☝️  no change then, but I still have to figure out when is the stylelint run, if run at all

Reply:
This should become a separate ticket.

Reply:
@boutell created: https://linear.app/apostrophecms/issue/PRO-2735/fix-stylelint-issues